schema/alert.xsd

```diff
@@ -0,0 +1,23 @@
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  attributeFormDefault="qualified"
+  elementFormDefault="qualified"
+  targetNamespace="https://hyperview.org/hyperview-alert"
+  xmlns:hv="https://hyperview.org/hyperview"
+>
+<xs:import namespace="https://hyperview.org/hyperview" schemaLocation="hyperview.xsd"/>
```

Comment:
I think `schemaLocation` might need to be changed to a real URI before we can publish this anywhere.

Reply:
yes, once we publish we'd want to replace it with a URL. But this works locally with `xmllint` so I'm leaving it for now. I think we'll need a publish command that replaces these locations with the global ones before pushing to production.